src/twisted/scripts/trial.py

```diff
@@ -608,7 +608,7 @@ def _makeRunner(config):
         from twisted.trial._dist.disttrial import DistTrialRunner
 
         cls = DistTrialRunner
-        args["workerNumber"] = config["jobs"]
+        args["maxWorkers"] = config["jobs"]
```

Comment:
I kept finding myself confused by "workerNumber" which is not the number of a worker but an upper limit on the number of worker processes to create.